pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/EventLogger.java

```diff
@@ -17,12 +17,18 @@
  */
 public class EventLogger {
 
-    private Var<LogEntry> latestEvent = Var.newSimpleVar(null);
+    private final Var<LogEntry> latestEvent = Var.newSimpleVar(null);
 
     public void logEvent(LogEntry event) {
         latestEvent.setValue(event);
     }
 
+    /**
+     * Gets a Stream of Log Events
```

Comment:
Remember documentation is here to explain what we can't readily guess from looking at the type. A good doc comment here, would explain concisely *when* the stream emits a new LogEntry